updateAssemblyInfo.ps1

```diff
@@ -43,7 +43,7 @@ Write-Host "assemblyInformationalVersion: "  $assemblyInformationalVersion
 $nugetSuffix = [string]$buildConfiguration.SelectSingleNode("root/nugetSuffix").InnerText
 if ( $packageType -eq "release")
 {
-    $versionSuffix = ""
+    $versionSuffix = $nugetSuffix
```

Comment:
nugetSuffix is typically set to preview in [buildConfiguration.xml]. By making this change, versionSuffix would be set to preview for release packages. Should Line 46 be reverted?

Reply:
do you mind taking care of this, @sruke?

Reply:
Set versionSuffix back to an empty string for release packages